be/src/exprs/bitmap_function.cpp

```diff
@@ -343,14 +367,19 @@ StringVal BitmapFunctions::bitmap_hash(doris_udf::FunctionContext* ctx, const do
 }
 
 StringVal BitmapFunctions::bitmap_serialize(FunctionContext* ctx, const StringVal& src) {
-    auto src_bitmap = reinterpret_cast<BitmapValue*>(src.ptr);
-    StringVal result = serialize(ctx, src_bitmap);
-    delete src_bitmap;
-    return result;
+    if (src.is_null) {
+        BitmapValue src_bitmap;
+        return serialize(ctx, &src_bitmap);
```

Comment:
I think is_null is diffrent from empty bitmap.
why here return an empty bitmap?

Reply:
I think `null` means "no data". And for "no data", return an empty set is reasonable.

Reply:
I also think there is no need to serialize null bitmap.
Set StringVal.is_null = true is better.
for in later merge function, we at first check if the StringVal is null.

Reply:
This serialize function is  multiplexed by finalize function in other bitmap function such as bitmap_union.